Mage.Sets/src/mage/cards/d/DeathInHeaven.java

```diff
@@ -0,0 +1,104 @@
+package mage.cards.d;
+
+import mage.abilities.Ability;
+import mage.abilities.common.SagaAbility;
+import mage.abilities.effects.Effects;
+import mage.abilities.effects.OneShotEffect;
+import mage.abilities.effects.common.ExileGraveyardAllTargetPlayerEffect;
+import mage.abilities.effects.common.MillCardsTargetEffect;
+import mage.abilities.effects.common.continuous.BecomePermanentFacedownEffect;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.cards.Cards;
+import mage.cards.CardsImpl;
+import mage.constants.*;
+import mage.game.Game;
+import mage.players.Player;
+import mage.target.TargetPlayer;
+import mage.util.CardUtil;
+
+import java.util.Optional;
+import java.util.UUID;
+
+/**
+ * @author TheElk801
+ */
+public final class DeathInHeaven extends CardImpl {
+
+    public DeathInHeaven(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.ENCHANTMENT}, "{3}{B}");
+
+        this.subtype.add(SubType.SAGA);
+
+        // (As this Saga enters and after your draw step, add a lore counter. Sacrifice after III.)
+        SagaAbility sagaAbility = new SagaAbility(this);
+
+        // I,II - Target player mills two cards, then exiles their graveyard.
+        sagaAbility.addChapterEffect(
+                this, SagaChapter.CHAPTER_I, SagaChapter.CHAPTER_II,
+                new Effects(
+                        new MillCardsTargetEffect(2),
+                        new ExileGraveyardAllTargetPlayerEffect(true)
+                                .setText(", then exiles their graveyard")
+                ), new TargetPlayer()
+        );
+
+        // III - Put all creature cards exiled with Death in Heaven onto the battlefield face down under your control. They're 2/2 Cyberman artifact creatures.
+        sagaAbility.addChapterEffect(this, SagaChapter.CHAPTER_III, new DeathInHeavenEffect());
+
+        this.addAbility(sagaAbility);
+    }
+
+    private DeathInHeaven(final DeathInHeaven card) {
+        super(card);
+    }
+
+    @Override
+    public DeathInHeaven copy() {
+        return new DeathInHeaven(this);
+    }
+}
+
+class DeathInHeavenEffect extends OneShotEffect {
+
+    private static final BecomePermanentFacedownEffect.PermanentApplier applier
+            = (permanent, game, source) -> {
```

Comment:
So does the face down status automatically set the power and toughness as 2/2 (and if so, where), or should it be set in this applier?

Reply:
yeah being face-down is automatically a creature with P/T 2/2 and no other features

Reply:
I don't know where though

Reply:
I've actually been looking into fixing morph lately and this shouldn't true according to the rules - 
`708.2. Face-down spells and face-down permanents have no characteristics other than those listed by 
the ability or rules that allowed the spell or permanent to be face down. Any listed characteristics 
are the copiable values of that object’s characteristics.`

Reply:
they specify it in [the following rule](https://yawgatog.com/resources/magic-rules/#R7082a)

Reply:
lol very true, ignore me